app/assets/stylesheets/pages/articles/article-list/_article-list.scss

```diff
@@ -0,0 +1,152 @@
+.article-list{
+	margin-top: 37px;
+
+	&__title{
+		font-size: 35px;
+		text-transform: capitalize;
+
+		@media only screen and (max-width: 1170px){
+			font-size: 33px;
+		}
+
+		@media only screen and (max-width: 640px){
+			font-size: 31px;
+		}
+	}
+
+	&__item{
+		display: block;
+		margin: 49px 0;
+	}
+
+	&__item-link{
+		text-decoration: none;
+
+		&:hover .article-list__banner-img{
+			opacity: .8;
+		}
+	}
+
+	&__banner-img{
+		background-image: image-url("bg.jpg");
+		background-size: cover;
+		height: 179px;
+		width: 275px;
+		display: inline-block;
+		border-radius: 15px;
+		transition: all .3s ease-in-out;
+
+		@media only screen and (max-width: 640px){
+			width: 40%;
+		}
+
+		@media only screen and (max-width: 480px){
+			width: 70%;
+			display: block;
+		}
+
+		@media only screen and (max-width: 410px){
+			width: 80%;
+		}
+
+		@media only screen and (max-width: 375px){
+			width: 100%;
+		}
+	}
+
+	&__txt-wrapper{
+		width: 40%;
+		display: inline-block;
+		vertical-align: top;
+		margin-left: 15px;
+
+		@media only screen and (max-width: 920px){
+			width: 45%;
+		}
+
+		@media only screen and (max-width: 768px){
+			width: 50%;
+		}
+
+		@media only screen and (max-width: 480px){
+			width: 70%;
+			display: block;
+			margin-left: 0;
+		}
+
+		@media only screen and (max-width: 410px){
+			width: 80%;
+		}
+
+		@media only screen and (max-width: 375px){
+			width: 100%;
+		}
+	}
+
+	&__item-title{
+		font-size: 20px;
+		margin: 0;
+		color: $defaultColor;
+
+		@media only screen and (max-width: 1170px){
+			font-size: 18px;
+		}
+
+		@media only screen and (max-width: 640px){
+			font-size: 16px;
+		}
+	}
+
+	&__description{
+		color: $textColorPrimary;
+		line-height: 1.3;
+		font-size: 18px;
+		margin-top: 5px;
+		margin-bottom: 0;
+
+		@media only screen and (max-width: 1170px){
+			font-size: 16px;
+		}
+
+		@media only screen and (max-width: 640px){
+			font-size: 14px;
+		}
+	}
+
+	&__author{
+		color: $textColorSecondary;
+		font-size: 16px;
```

Comment:
it's the default size, so kind of useless if you describe it, remove it

Reply:
okok